shared/reporter/collect.go

```
@@ -0,0 +1,222 @@
+}
+
+func (c *collector) statusOK(ctx context.Context, url string) bool {
+	request, err := http.NewRequestWithContext(ctx, http.MethodGet, url, nil)
+	if err != nil {
+		return false
+	}
+	response, err := c.client.Do(request)
+	if err != nil {
+		return false
+	}
+	defer response.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `response.Body.Close` is not checked

(errcheck)